src/Components/Title/index.js

```diff
@@ -0,0 +1,9 @@
+import React, {Component} from "react"
+import "./index.css"
+
+export default class Title extends React.Component {
+    render(){
+        return <h1 className="Title">{this.props.propTitle}</h1>
```

Comment:
I would change this to use `this.props.children` instead. This way we can use this component with more complex html strucutres if needed. Please take a look [here](https://mxstbr.blog/2017/02/react-children-deepdive/) for more information about `this.props.children`